### ISSUE TITLE
passThrough no longer works when a relative baseURL is defined

### ISSUE THREAD

**Issue by jrparish** [ctimmerm/axios-mock-adapter#185]
A recent change (https://github.com/ctimmerm/axios-mock-adapter/pull/146) breaks `passThrough` when a relative `baseURL` is defined. It now prepends the baseURL twice when performing the pass through.

I wrote a test to demonstrate the failure - https://github.com/jrparish/axios-mock-adapter/blob/feature/pass-through-test/test/pass_through.spec.js#L113

**Comment by barakbd-bluevine** [comment]
Passthrough is not working even with a full baseURL.
```
const axiosService = Axios.create({
  baseURL: 'http://localhost:8900/api',
  timeout: 30000,
  withCredentials: true,
  xsrfHeaderName: 'X-CSRFToken',
  xsrfCookieName: 'csrftoken',
  headers: {
    'Content-Type': 'application/json'
  }
})
```
Any suggestion for a quick fix so I can use it now?

**Comment by barakbd-bluevine** [comment]
actually, I am not seeing any network requests with passthrough.
I have no mock APIs setup, and it is still not working. 
```mock.onAny().passThrough()```

**Comment by jrparish** [comment]
You can handle it in an interceptor, but in our case we were depending on the old behavior for logging, so I just reverted to `1.15.0`.

**Comment by rodmax** [comment]
this issues is reproduced in my project too,

i implemented simple interceptor to workaround this

```typescript
axiosInstance.interceptors.request.use(config => {
    if (config.baseURL && config.url.startsWith(config.baseURL)) {
        config.url = config.url.slice(config.baseURL.length);
    }
    return config;
});
```

**Comment by michael-ecb** [comment]
any updates ? 
thanks

**Comment by ctimmerm** [comment]
Fixed by #200, added a test case for it: https://github.com/ctimmerm/axios-mock-adapter/blob/master/test/pass_through.spec.js#L145
